lang/en.json

```
@@ -887,6 +887,9 @@
         "TimeSinceLoadVariableUnitTitle": "Unit",
         "TimeSinceLoadVariableUnitDescription": "Select in which unit you want to retrieve the time.",
         "UrlParameterVariableNameTitle": "URL Parameter Name",
-        "UrlParameterVariableNameDescription": "For example when your page has URL parameters such as \"?lang=NZ\" and you want to get the value \"NZ\", then you need to enter \"lang\"."
+        "UrlParameterVariableNameDescription": "For example when your page has URL parameters such as \"?lang=NZ\" and you want to get the value \"NZ\", then you need to enter \"lang\".",
+        "UserInteractionTriggerName": "User Interaction",
+        "UserInteractionTriggerDescription": "Triggered when user interacts with the site, i. e. touch, move, scroll or mouse click event is executed by the user on site.",
+        "UserInteractionTriggerHelp": "Triggered when any of the following event touch,move,scroll or mouse click is executed by the user on site."
```

@JaJuMa Thanks for all the changes the PR looks good now
```suggestion
        "UserInteractionTriggerHelp": "Triggered when any of the following event touch, move, scroll or mouse click is executed by the user on site."
```

---

Altamash's suggested change of adding spaces should probably be applied as well as updating [line 420 of tests/System/expected/test_webContext__TagManager.getAvailableTriggerTypesInContext.xml](https://github.com/matomo-org/tag-manager/blob/b1bad3fd93ccfddc2312651ca5f01029e8028132/tests/System/expected/test_webContext__TagManager.getAvailableTriggerTypesInContext.xml#L420) to match.